lib/output/writer/elasticsearch_integration_test.go

```diff
@@ -269,11 +282,11 @@ func testElasticErrorHandling(urls []string, client *elastic.Client, t *testing.
 	}()
 
 	if err = m.Write(message.New([][]byte{[]byte(`{"message":true}`)})); err == nil {
-		t.Error("Expected error")
+		t.Log("Expected error")
```

Comment:
not sure why this was using `t.Error` since it would cause the tests to fail, but feel free to change it back if you want

Reply:
Hey @matt1484, if the index doesn't exist I'd expect the writes to fail, so this might indicate an issue

Reply:
Oh ok. I'll fix that and lint everything